scripts/devaccess/devaccess_users.yaml

```diff
@@ -1,14 +1,179 @@
+---
 roles:
 - name: ALL
   commands:
-  - oc get nodes -ndefault
-  - oc get nodes -ndefault -ojson
-  - rpm -qa
-  - oc get pods -ndefault
-  - oc get pods -nopenshift-infra
-  - oc get pods -nlogging
-  - oc get routes -ndefault
-  - oc get routes -nlogging
+    - base: oc get
+      types:
+        - names:
+            - node
+            - nodes
+          runner: oc_get_generic
+          output:
+            - json
+        - names:
+            - pod
+            - pods
+          runner: oc_get_generic
+          namespaces:
+            - default
+            - logging
+            - openshift-infra
+        - names:
+            - route
+            - routes
+          runner: oc_get_routes
+          namespaces:
+            - default
+            - logging
+        - names:
+            - storageclass
+          runner: oc_get_generic
+        - names:
+            - pvc
+          runner: oc_get_generic
+          namespaces:
+            - all
+        - names:
+            - ns
+            - project
+            - projects
+            - namespace
+            - namespaces
+          runner: oc_get_generic
+        - names:
+            - svc
+            - service
+            - services
+          runner: oc_get_generic
+          namespaces:
+            - all
+
+    - base: oc describe
+      types:
+        - names:
+            - clusterresourcequota
+
+    - base: rpm
+      runner: safe_command
+      switches:
+        - -qa
+
+    - base: cat
+      runner: super_safe_command
```

Comment:
cat is very much NOT a "safe command" unless filenames remain tightly controlled.  If anything in this system fails in the future, I would put money on unintentional loosening of controls around a command like this - I think it's worth adding comments or other protections to help prevent.